src/components/ProfileInformation/ProfileInformation.tsx

```diff
@@ -6,29 +6,37 @@ import Copy from '../../assets/icons/Copy.svg'
 import People from '../../assets/icons/People.svg'
 import Share from '../../assets/icons/Share.svg'
 import Wallet from '../../assets/icons/Wallet.svg'
+import { Props } from './ProfileInformation.types'
 import styles from './ProfileInformation.module.css'
 
-const ProfileInformation = () => {
-  return (
+const ProfileInformation = (props: Props) => {
+  const { profile, isLoggedInProfile = false } = props
```

Comment:
If `isLoggedInProfile` is not optional, we should remove the default value, right?
```suggestion
  const { profile, isLoggedInProfile = false } = props
```